### ISSUE TITLE
Needs less strict address format requirements

### ISSUE THREAD

**Issue by fcFn** [LssPolymerElements/titanium-address-input#6]
I'm trying to get the address via `this.$.addr.address` (`addr` is the `titanium-address-input` id) after selecting an address from the list of suggestions, but I only get `null` back. Am I doing something wrong?

**Comment by fcFn** [comment]
While trying to figure this out, I found out that any address without a zip-code (i. e. any non-US address?) will cause the address to be set to `null`.

**Comment by antwonthegreat** [comment]
That is correct. The way it is currently written, it will only accept an address that has all of: administrative_area_level_1 (state), street_number, route (street name), locality (city), and postal_code. Might need to add a property that can be set to allow for less restrictive addresses.

**Comment by fcFn** [comment]
Thanks. I've modified the component for my use case for now (won't be suitable for PR as I removed all the filters from `_onChangePlace` and added the filtering to `_formatAddress` to produce a proper value).

By the way, the [docs](https://www.webcomponents.org/element/LssPolymerElements/titanium-address-input/elements/titanium-address-input) might need fixing, as it says this for `value` property which actually contains the formatted address string:
```
value: string
Required. Google Maps API Key.
```
And there is no description for `api_key` prop.
